python/paddle/fluid/layers/tensor.py

```diff
@@ -327,6 +327,10 @@ def concat(input, axis=0, name=None):
     out = helper.create_variable_for_type_inference(dtype=helper.input_dtype())
 
     if input[0].desc.type() == core.VarDesc.VarType.LOD_TENSOR_ARRAY:
+        # NOTE(liym27): Don't remove this if branch!
+        # This feature is supported for Dynamic-to-Static, because after transformed, the type of inputs[0] is LOD_TENSOR_ARRAY in some scenarios.
+        # But it's not recommended for users to use this feature in static mode.
```

Comment:
Delete "not recommended for users". In my understanding, user still needs TensorArray at some time.

Reply:
Thanks, use "And this feature can be used in static mode." to replace " But it's not recommended for users to use this feature in static mode."